week2/index.html

```diff
@@ -0,0 +1,47 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title >My First HTML</title>
+        <nav class="menu">
+            <ul>
+              <li><a href="#">Home</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </nav>
+    </head>
+    <header>
+        <h1>My First HTML Page HTML Elements</h1>
+    </header>
+    <body>
+        <section>
+            <h2><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element" title="The best place to find more information about Mozilla's
+                mission and how to contribute" target="_blank">HTML Elements</a></h2>
+        </section>
+        <p>
```

Comment:
The section element is a container for content that is related. Hence, this &lt;p&gt; should be part of above &lt;section&gt;